tasks/mc/data.py

```diff
@@ -392,9 +392,14 @@ def _insert_tr(self, session):
                     FROM
                         "{input_schema}".{input_table} mc
                     WHERE mc.category = 'total retail'
-                    AND (mc.{region_id}, mc.{month}) NOT IN
-                      (select {region_id}, {month}
-                       from "{output_schema}".{output_table});
+                    AND NOT EXISTS
```

Comment:
The new query looks OK to me. Should work fine for bigger sizes as long as we have an index.

I'm curious if this is better or worse than using the `ON CONFLICT` clause for insert, e.g:
```sql
INSERT INTO wadus (region, month, fields)
SELECT mc.region, mc.month, fields
FROM input_table
ON CONFLICT (region, month) DO NOTHING
```

This moves the check to the insertion rather than selection. I guess it depends on the size of the input and output table. Assuming that output is bigger than input, it's probably worse.

Reply:
I'll give this a try, but this change makes USA be updated very fast (blocks take 2 min, while the previous code stalled at tracts), so it's not a big deal.

Reply:
I wrote this to feel cool (really trying to learn if the `ON CONFLICT` syntax could be used here). If the current approach is fast, this is not likely to be faster. Try if you like, but otherwise, let's merge.

Reply:
Blocks, with `NOT EXISTS`: 122s
Blocks, with `ON CONFLICT` (replacing it not only in the first `INSERT` but also in the category `UPDATE`): 150s.

So, it's probably equivalent, although not faster. It probably implies row-based checks instead of a big select and filter, index-based, so it makes sense. I'll keep the `EXISTS` one.

_PS: don't merge yet, I'll also push the partial dump script for production update._